code/modules/reagents/Chemistry-Logging.dm

```diff
@@ -18,7 +18,7 @@
 	set name = "Show Chemical Reactions"
 	set category = "Admin"
 
-	if(!check_rights(R_ADMIN))
+	if(!check_rights(R_ADMIN|R_MOD))
```

Comment:
For some reason it's | and not || in dm, should work as intended as coded.

On Sun, Nov 29, 2015, 15:39 Neerti <notifications@github.com> wrote:

> In code/modules/reagents/Chemistry-Logging.dm
> <https://github.com/PolarisSS13/Polaris/pull/355#discussion_r46097658>:
>
> > @@ -18,7 +18,7 @@
> >  	set name = "Show Chemical Reactions"
> >  	set category = "Admin"
> >
> > -	if(!check_rights(R_ADMIN))
> > +	if(!check_rights(R_ADMIN|R_MOD))
>
> Should this be || ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PolarisSS13/Polaris/pull/355/files#r46097658>.
>